src/composables/maxAmount.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  ref,
+  computed,
+  onMounted,
+  watch,
+} from '@vue/composition-api';
+import { pick } from 'lodash-es';
+import BigNumber from 'bignumber.js';
+import { Store } from 'vuex';
+import FUNGIBLE_TOKEN_CONTRACT from 'aeternity-fungible-token/FungibleTokenFullInterface.aes';
+import { TxBuilder, SCHEMA } from '@aeternity/aepp-sdk';
+import { MAGNITUDE, STUB_CONTRACT_ADDRESS, AETERNITY_CONTRACT_ID } from '../popup/utils/constants';
+import { rxJsObservableToVueState } from '../popup/utils/utils';
+
+import {
+  executeAndSetInterval,
+  calculateFee,
+  validateTipUrl,
+  checkAensName,
+  handleUnknownError,
+  watchUntilTruthy,
+} from '../popup/utils';
+import { IAccount } from '../types';
+
+export interface UseMaxAmountOptions {
+  store: Store<any>
+  formModel: any,
+}
+
+export function useMaxAmount({ store, formModel }: UseMaxAmountOptions) {
+  const fee = ref(new BigNumber(0));
+  const selectedTokenBalance = ref(new BigNumber(0));
+  const tokenInstance = ref<any>(null);
+  const nonce = ref(0);
+  const selectedAssetDecimals = ref(0);
+
+  const balance = rxJsObservableToVueState(store.state.observables.balance);
+  const account = computed<IAccount>(() => store.getters.account);
```

Comment:
on https://github.com/aeternity/superhero-wallet/pull/1706 i created useGetter for get somethink from store

Reply:
it's an observable's value, cannot use getters for it

Reply:
@ifaouibadi the account is not related to observables (RxJS). So I agree with Konrad on this.